src/utils/worklets/shell.cljs

```diff
@@ -59,6 +59,10 @@
   [home-stack-state-sv top-home-stack-position]
   (.homeStackTop ^js home-stack-worklets home-stack-state-sv top-home-stack-position))
 
+(defn home-stack-border-radius
+  [home-stack-state-sv]
```

Comment:
what is `sv`?

Reply:
https://github.com/status-im/status-mobile/pull/15247#discussion_r1125021144

Reply:
changed, thank you for suggestion